2-alcance/README.md

```diff
@@ -1,3 +1,112 @@
+
+Escenario 2: Fran es un fan incondicional de los grupos rock de todo el mundo, y de toda la historia. Es un mecánico en un concesionario de su ciudad, y cuando puede, en el trabajo o en su caso siempre suele escuchar música rock a todo volumen.  Le gusta salsear para buscar grupos de este estilo que no son tan famosos o que son principiantes, para escucharlos y degustar otras canciones que entre la gente no son conocidas, pero que son buenos. 
+
+Fran está suscrito a la página web de Musikamania.com para poder satisfacer sus intereses, y conocer nuevos grupos que no son tan ‘top’. También utiliza esta página web para comentar con otros usuarios las canciones de los grupos, o compartirlas mediante sus redes sociales. 
+
+
+Escenario 3: Oscar es un joven madrileño aficionado a la música y sobre todo a conciertos y festivales. Suele acudir a ellas con sus amigos o sus parejas. Le gusta comprar con antelación las entradas, para que luego no haya problemas para poder acudir a ellas. 
+
+Está dentro de un grupo grande de Facebook de aficionados a los festivales, y en ella ha encontrado una noticia publicada en la página web de Musikamania.com, sobre las fechas de todo el año de festivales y venta de entradas para poder comprarlas. Oscar ha llegado a la página web gracias a que otro usuario haya compartido esa noticia en Facebook, y a él le ha parecido interesante. Oscar está interesado en suscribirse la página web para enterarse sobre los festivales de toda España  y comprar entradas. 
+
+### No objetivos 
+
+Esta versión no incluirá las siguientes funciones:
+
+- Descargar música.
+- Comprar CDs de los grupos.
+
+### Detalles
+
+Musikamania.com tendrá tres tipos de usuarios y cada tipo tendrá sus funciones específicas. 
+
+Administradores: serán los encargados de llevar a cabo la página web. Su función será editar, configurar e incluso diseñar los contenidos que se ofrecerán y la página web en general. También se encargarán de controlar las visitas que tiene la página para saber si está funcionando bien o hay que hacer algún cambio.
+
+Persona no suscrita: serán todas aquellas personas que accedan a la página web sin ser suscritas. Estas personas podrán acceder únicamente a algunos de los contenidos que se ofrezcan en la página. 
+
+Persona suscrita: estas personas, a parte de tener todas las funciones de las personas no suscritas, también tendrán otras funciones exclusivas para ellas:
+
+- Podrán recibir noticias sobre los géneros musicales que hayan elegido los usuarios a la hora de registrarse.
+- Opción a ver más contenido, contenido más desarrollado.
+- Dejar comentarios debajo de cualquier contenido, ya sea, noticia, entrevista, reportaje etc. 
+- Compartir los contenidos a través de las redes sociales como Twitter o Facebook.
+- Escuchar las listas de canciones que aparezcan en la página web. Las personas no registradas también podrán acceder a las listas de canciones, pero no las podrán escuchar.
```

Comment:
- Publicar contenidos básicamente textuales (noticias, entrevistas, reportajes)
- Publicar listados de canciones
- Reproducir listados de canciones (limitado a suscriptores)
- Limitar el acceso a algunos tipos de contenido

Son todas tareas que debéis definir individualmente como tales, y especificar algo más en qué consiste cada una.